templates/search/blocks/search_results_hero.html

```diff
@@ -8,7 +8,7 @@ <h1 class="search-results-hero__heading">{{search_tab|search_title}}</h1>
         {{ form.q }}
         
         {% if search_tab == searchtabs.CATALOGUE.value or search_tab == searchtabs.WEBSITE.value %}
-            {{ 'q'|include_hidden_fields:form }}
+            {{ 'catalogue-search-form'|include_hidden_fields:form }}
```

Comment:
I've mentioned this API already. Just highlighting here so that it isn't missed.

Reply:
Same comment in relation to `hidden_fields_for_date_filter` and `bound field methods`

Reply:
`{% render_form_fields_as_hidden form=form exclude='q' %}` would be much better